src/main/java/com/google/devtools/build/lib/analysis/starlark/Args.java

```diff
@@ -70,7 +72,8 @@ public void repr(Printer printer) {
   @Override
   public void debugPrint(Printer printer, StarlarkSemantics semantics) {
     try {
-      printer.append(Joiner.on(" ").join(build().arguments()));
+      printer.append(
+          Joiner.on(" ").join(build(/* mainRepoMappingSupplier */ () -> null).arguments()));
```

Comment:
Need equals sign on parameter comment.

Reply:
this doesn't seem resolved?

Reply:
Fixed, I accidentally reverted this together with the previous approach.